src/FilamentPasswordlessPlugin.php

```diff
@@ -0,0 +1,56 @@
+<?php
+ 
+namespace BradyRenting\FilamentPasswordless;
+ 
+use BradyRenting\FilamentPasswordless\Http\Livewire\Auth\Login;
+use Filament\Contracts\Plugin;
+use Filament\Panel;
+use Illuminate\Foundation\Auth\User as Authenticatable;
+ 
+class FilamentPasswordlessPlugin implements Plugin
+{
+    protected string $model;
+
+    public function __construct()
+    {
+        $this->model = config('filament-passwordless.model');
+    }
+
+    public static function make(): static
+    {
+        return app(static::class);
+    }
+
+    public function getId(): string
+    {
+        return 'filament-passwordless';
+    }
+ 
+    public function register(Panel $panel): void
+    {
+        $panel
+            ->authGuard(env('FILAMENT_AUTH_GUARD', 'web'))
```

Comment:
This will override any existing guard already configured on the panel. Filament already defaults to the configured auth guard so this shouldn't be necessary.

(Also, you should only use env(...) in config files, as this could otherwise cause issues with config caching)